src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs

```diff
@@ -193,10 +193,15 @@ internal IEnumerable<FunctionMetadata> ValidateMetadata(IEnumerable<RawFunctionM
                         }
                     }
 
-                    // retry option validation
+                    // populate retry options if json string representation is provided
                     if (!string.IsNullOrEmpty(rawFunction.RetryOptions))
                     {
                         function.Retry = JObject.Parse(rawFunction.RetryOptions).ToObject<RetryOptions>();
```

Comment:
Oh, we still have the Json string for the retry option data ? I See this is existing code. Are we planning to make this prop obsolete and recommend language owners to use the strongly typed version (The new property we recently added to proto file)?

Reply:
The strongly typed version added here is only for worker-indexing scenarios - I'm not sure if host indexing (or other providers like extension loader or custom providers from partner services) are still using the legacy bits.